src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import { HStack, Flex, FlexProps } from "@chakra-ui/react"
+import { PropsWithChildren } from "react"
+
+export const Footer = ({
+  children,
+  styles,
```

Comment:
what we can do here is to have a rest parameter `...rest` and spread it after; this is so that the API stays identical to `Flex`

Reply:
https://github.com/isomerpages/isomercms-frontend/pull/932/commits/c62f356a185d6ef76426aee118446b04d78f9e46